src/main/java/cuchaz/enigma/analysis/index/JarIndex.java

```diff
@@ -58,18 +60,18 @@ public static JarIndex empty() {
 	}
 
 	public void indexJar(ClassCache classCache, ProgressListener progress) {
-		progress.init(4, "Indexing jar");
+		progress.init(4, LangUtils.translate("progress.jar"));
 
```

Comment:
how about naming the keys by the process, it might be bad if we just suddenly add another process in the middle. Naming them by step index is kinda bad